### ISSUE TITLE
I/O functions implementation from/to console and file

### ISSUE THREAD

**Issue by sergioferrari** [bernedom/SI#13]
**Obviously, the users of SI library, needs to input values at runtime and view outputs from applications**

**In my opinion, if I use a certain physical unit in a determinate scale, I want to see (for default) the variable's value in the scale I chose, with the related symbol**

**I am a C ++ amateur and certainly I cannot afford to offer you solutions, especially about generic programming, with extensive use of templates**

**Comment by bernedom** [comment]
Have a look at the current master or the [1.3.0 unstable conan package on bintray](https://bintray.com/bernedom/conan/SI%3ASI/1.3.0%3Aunstable) 
Streaming to the test/console/files are there. 

example:
```cpp
constexpr auto value = 1_km;
  std::stringstream ss;
  ss << value;   // ss now contains "1km"
```

Reading from streams is [currently being developed](https://github.com/bernedom/SI/tree/feature/instream-operator)

**Comment by bernedom** [comment]
Available from version 1.3.0 on
